src/recipe.rs

```diff
@@ -69,8 +69,15 @@ impl<'src, D> Recipe<'src, D> {
   }
 
   pub(crate) fn confirm(&self) -> RunResult<'src, bool> {
-    if self.attributes.contains(&Attribute::Confirm) {
-      eprint!("Run recipe `{}`? ", self.name);
+    if let Some(Attribute::Confirm(prompt_text)) = self
```

Comment:
I think we should probably completely replace the prompt, so that users can customize it however they want. I.e., we either print out the default prompt if there's none given, or we print the user-specified prompt, and nothing else. This allows users fully customize the prompt.

Reply:
That seems better yes.